drivers/node/ssh/ssh.go

```diff
@@ -746,7 +746,8 @@ func (s *SSH) GetBlockDrives(n node.Node, options node.SystemctlOpts) (map[strin
 			Cause: err.Error(),
 		}
 	}
-	driveOutput := fmt.Sprint(out)
+	driveOutput := fmt.Sprintf("%s", out)
+	fmt.Println(out)
```

Comment:
Lets use `log` here instead of `fmt` or remove it if its for debugging purposes

Reply:
remove the line